client/app/components/viz/VariantInspectCard.vue

```diff
@@ -1258,6 +1260,11 @@ export default {
   },
   methods: {
 
+    updateMosaicVariantObject(mosaicVariant) {
```

Comment:
The function name 'update...' is a bit misleading as we are not saving the variant back to Mosaic, but just initializing this field. Consider renaming to 'initializeMosaicVariantObject' or 'initializeMosaicVariant' . (leave out unecessary terms like 'Object' as it lengthens method name.